mrack.spec

```diff
@@ -12,48 +12,120 @@ Source0:        https://github.com/neoave/mrack/releases/download/v%{version}/mr
 BuildArch:      noarch
 
 BuildRequires:  python3-devel
-BuildRequires:  python3-AsyncOpenStackClient
-BuildRequires:  beaker-client
-BuildRequires:  python3-boto3
-BuildRequires:  python3-botocore
 BuildRequires:  python3-click
 BuildRequires:  python3-pyyaml
 BuildRequires:  python3-setuptools
 
+# coma separated list of provider plugins
+%global provider_plugins aws,beaker,openstack,podman,virt
+
 %description
 mrack is a provisioning library for CI and local multi-host testing supporting multiple
-provisioning providers (e.g. OpenStack, Beaker). But in comparison to other multi-cloud
+provisioning providers (e.g. AWS, Beaker, Openstack). But in comparison to other multi-cloud
 libraries, the aim is to be able to describe host from application perspective.
 
-%{?python_provide:%python_provide %{srcname}}
+%{?python_provide:%python_provide %{name}}
 
-Requires:       python3-AsyncOpenStackClient
-Requires:       beaker-client
-Requires:       podman
-Requires:       python3-boto3
-Requires:       python3-botocore
+Requires:       %{name}-core
+Requires:       %{name}-aws
+Requires:       %{name}-beaker
+Suggests:       %{name}-openstack
```

Comment:
This isn't needed IMO, the `mrack-openstack` sub-package is installable because *that* package uses "only" Suggests for problematic deps.

Reply:
So at this level `Requires` is safe correct?